cross-chain/optimism/deploy_l2/12_update_wormhole_gateway_mapping.ts

```diff
@@ -0,0 +1,29 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, ethers } = hre
+  const { execute } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // See https://book.wormhole.com/reference/contracts.html
+  // This ID is valid for both Optimism Goerli and Mainnet
+  const wormholeChainID = 24
+
+  const optimismWormholeGateway = await deployments.get(
+    "OptimismWormholeGateway"
+  )
+
```

Comment:
Let's also map the previously deployed Arbitrum Gateway.

For Arbitrum Gateway, let's make sure we have a checkpoint in the Optimism deployment scenario to set the mapping for Optimism gateway in Arbitrum gateway. This will have to be done by the Threshold Council.

Reply:
Added mapping of the Arbitrum Wormhole Gateway [8394151](https://github.com/keep-network/tbtc-v2/pull/585/commits/83941510c6ea312d7476285093dfdd5502c79d6a)

Reply:
I also think hardcoding is a better option in this case. It is just simpler and does not lead to dependency hell long-term.